public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -553,21 +554,21 @@ block dart-type-exception-example
 a(id="one-time-initialization")
 :marked
   ### One-time string initialization
-  We *should* omit the brackets when all of the following are true:
+  The brackets *should* be omited when the following conditions are true:
   * The target property accepts a string value.
   * The string is a fixed value that we can bake into the template.
   * This initial value never changes.
 
-  We routinely initialize attributes this way in standard HTML, and it works
+  This initialization is routinely used in standard HTML, and it works
```

Comment:
Again, avoid passive voice. Maybe:

Initializing attributes this way is common in standard HTML, and...